loaders/src/glTF/2.0/Extensions/KHR_materials_transmission.ts

```diff
@@ -73,6 +327,9 @@ export class KHR_materials_transmission implements IGLTFLoaderExtension {
 
         if (extension.transmissionFactor !== undefined) {
             pbrMaterial.subSurface.refractionIntensity = extension.transmissionFactor;
+            if (pbrMaterial.subSurface.refractionIntensity && this._transmissionHelper == undefined) {
+                this._transmissionHelper = new TransmissionHelper({}, pbrMaterial.getScene());
```

Comment:
I'm not sure what would keep this object alive. This loader extension should, in theory, get garbage collected once the load is complete. Once that happens, who is holding this object? Is that an issue?

Reply:
Yes, this is an issue with doing things this way. The helper should ideally stick around so that it can handle material changes and mesh additions and removals. Also, we shouldn't be creating new helpers for each transmissive glTF loaded... that would be bad.

Reply:
Okay, finally had a chance to change this.